src/theme/material/assets/nohandler.scss

```diff
@@ -0,0 +1,115 @@
+$duration: 0.15s;
```

Comment:
can we put it under existing focus.scss?

Reply:
If I change focuss.scss, it will affect the whole qgrid. As I understood, we need it only in Action Bar On Enter. Is it right?